docs/writing-helmfile.md

```diff
@@ -89,6 +89,42 @@ releases:
   <<: *default
 ```
 
+Release Templating supports the following parts of release definition:
+- basic fields: `name`, `namespace`, `chart`, `version`
+- boolean fields: `installed`, `wait`, `tillerless`, `verify` by the means of additional text
+  fields designed for templating only: `installedTemplate`, `waitTemplate`, `tillerlessTemplate`, `verifyTemplate`
+  ```yaml
+  # ...
+    installedTemplate: '{{`{{ eq .Release.Namespace "kube-system" }}`}}'
+    waitTemplate: '{{`{{ eq .Release.Labels.tag "safe" | not }}`}}'
+  # ...
+  ```
+- `set` block values:
+  ```yaml
+  # ...
+    set:
```

Comment:
Accordingly to https://github.com/roboll/helmfile/pull/823#issuecomment-526800928, I'd be happy if we could enable templates only for `setTemplate` and `valuesTemplate` rather than `set` and `values` respectively.

I'll make the changes on my own before releasing this, but if you made it on your side I'd be happy to review/accept the PR! Thanks.

Reply:
See #833

Reply:
Merged #833. Please feel free to leave any comments about it!